lib/client.js

```diff
@@ -124,19 +124,21 @@ class Client {
    * @example
    *
    * // Using promises
-   * fhirClient.read('Patient', 12345).then((data) => { console.log(data); });
+   * fhirClient.read({ resourceType: 'Patient', identifier: 12345 })
+   *           .then((data) => { console.log(data); });
    *
    * // Using async
-   * let response = await fhirClient.read('Patient', 12345);
+   * let response = await fhirClient.read({ resourceType: 'Patient', identifier: 12345 });
    * console.log(response);
    *
-   * @param {string} resource The resource type (e.g. "Patient", "Observation")
-   * @param {string} identifier The FHIR identifier for the resource
+   * @param {Object} params - The request parameters.
+   * @param {string} params.resourceType - The resource type (e.g. "Patient", "Observation").
+   * @param {string} params.identifier - The FHIR identifier for the resource.
```

Comment:
I think this should be `id` rather than `identifier` for all of these, since these are FHIR `id`s rather than FHIR `identifier`s.

Reply:
yea, I agree `identifier` is something else in FHIR